### DISCUSSION TITLE
BIQU HDMI 7 add resolution 1024x600

### DISCUSSION THREAD

**Discussion by HunLucas** [rern/rAudio#1192] [Fixed Issues] [answered]
Hello everyone!

I have a BIQU HDMI Touchscreen, its running only in 640x480 on my rpi4 with rAudio.
if I add manually 1024x600 resolution into a config file in xorg.conf.d folder, the chromium wont start.
xrandr command give me:  "can't open display" error code

Rern, can you add custom resolution in your next update?
Thank You

**Comment by HunLucas** [comment, marked as answer]
I added this lines into boot/config.txt
```
# uncomment this if your display has a black border of unused pixels visible
# and your display can output without overscan
#disable_overscan=1

# uncomment the following to adjust overscan. Use positive numbers if console
# goes off screen, and negative if there is too much border
overscan_left=6
overscan_right=333
overscan_top=2
overscan_bottom=180

# uncomment to force a console size. By default it will be display's size minus
# overscan.
framebuffer_width=1024
framebuffer_height=600
```
Then touch calibration:
```
export DISPLAY=:0
xinput_calibrator | tail -6 > /usr/share/X11/xorg.conf.d/99-calibration.conf
```
Thanks to everyone who helped me, well... no one helped me...
I hope this help for someone else

sza rja tok sünt Béláim!

**Comment by philzilla** [comment]
thank you !

**Reply by HunLucas** [reply]
It is working:

hdmi_force_hotplug=1
hdmi_cvt=1024 600 60
hdmi_group=2
hdmi_mode=87
hdmi_pixel_encoding=2
disable_overscan=1
#overscan_left=-2
#overscan_right=-30
#overscan_top=-2
#overscan_bottom=-30
framebuffer_width=1024
framebuffer_height=600
